compiler/z/codegen/S390GenerateInstructions.cpp

```diff
@@ -3211,3 +3211,11 @@ template TR::Instruction * generateS390CompareAndBranchInstruction<int64_t>(
                     bool targetIsFarAndCold = false,
                     TR::Instruction        *preced = 0,
                     TR::RegisterDependencyConditions *cond = 0); 
+
+TR::Instruction*
+generateAlignmentNopInstruction(TR::CodeGenerator *cg, TR::Node *node, uint32_t alignment, TR::Instruction *preced)
+   {
+   if (preced)
+      return new (INSN_HEAP) TR::S390AlignmentNopInstruction(node, alignment, preced, cg);
+   return new (INSN_HEAP) TR::S390AlignmentNopInstruction(node, alignment, cg);
+   }
```

Comment:
Weird EOL.

Reply:
Fixed in [force-push](https://github.com/eclipse/omr/compare/aa4d2cf9e7c421a5ea39e9f8d701fd14e58ddbca..0b3376f0ba6f553545daf6908b91dc0a9729d50e).